src/android/CHIPTool/app/src/main/res/layout/enter_thread_network_fragment.xml

```diff
@@ -1,121 +1,113 @@
 <?xml version="1.0" encoding="utf-8"?>
-<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Would it be possible to keep ScrollView here? On my old smartphone the form occupies almost the entire screen so that the button obscures part of the Master Key edit box.

Reply:
Done.